includes/legacy/class-wc-amazon-payments-advanced-api-legacy.php

```diff
@@ -574,23 +593,29 @@ public static function get_capture_state( $order_id, $id ) {
 	public static function get_order_ref_state( $order_id, $state = 'amazon_reference_state' ) {
 		$ret_state = '';
```

Comment:
Can we improve the name of this variable, please?

Reply:
done!